From 8a9e6d4c98e75d8ad542958db8ae4a29f75cb2d9 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Fri, 26 Jan 2024 09:10:48 +0100
Subject: [PATCH] Link lz4 in test_nearestneighbors

---
 tests/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 77f47efc39..bfb05d8b13 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -18,7 +18,9 @@ if (OMPL_BUILD_TESTS)
     add_ompl_test(test_gridb datastructures/gridb.cpp)
     add_ompl_test(test_nearestneighbors datastructures/nearestneighbors.cpp)
     if(flann_FOUND)
-        target_link_libraries(test_nearestneighbors ${FLANN_LIBRARIES})
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(LZ4 REQUIRED liblz4)
+        target_link_libraries(test_nearestneighbors ${FLANN_LIBRARIES} ${LZ4_LINK_LIBRARIES})
         # FLANN uses std::random_shuffle in a header file which is removed in C++17
         set_target_properties(test_nearestneighbors PROPERTIES CXX_STANDARD 14)
     endif()
